test/systemtest/conftest.py

```diff
@@ -133,6 +133,14 @@ def logfile(pytestconfig):
     log = pytestconfig.getoption('log')
     if not log:
         return
-    path = Path(log).resolve()
-    assert not path.is_dir()
+    # The strict option is only availabe on pythong 3.6
+    # CI currently uses 3.5
```

Comment:
This is true, but more accurate to say that "However, Python 3.5.2 is our minimum supported version."

I think this should be pulled out to separate patch and could even be pulled out to a separate PR as it's a worthwhile fix even without the riscv compliance context.